gulpfile.js

```diff
@@ -110,13 +110,20 @@ function createWebpackConfig(defines, output) {
     module: {
       loaders: [
         {
+          test: /(\bweb\b).*?\.js$/,
```

Comment:
Right, I forgot to change that to add 'src'. On the second thought, enabled for global (not limited to the web folder) will work fine too. Removing the line.